app/validators/answers/number_answer_validator.py

```diff
@@ -58,6 +70,20 @@ def validate_mandatory_has_no_default(self):
         if self.answer.get("mandatory") and self.answer.get("default") is not None:
             self.add_error(self.DEFAULT_ON_MANDATORY)
 
+    def validate_min_max_is_number(self):
+        for min_max in ["minimum", "maximum"]:
+            value = self.answer.get(min_max, {}).get("value", 0)
+            if value:
```

Comment:
You can compress these two lines into
`if value := self.answer.get(min_max, {}).get("value"):`
(no need to default to 0 in this case)